### PR TITLE
New plugin examples

### PR DESCRIPTION
**Pull request by mediocretes**
All of the things monitored by plugin scripts in instrumental_tools are now monitored by telegraf, so, we need new examples to demonstrate the value of plugin scripts.

This PR removes the documentation and adds one (so far) example of a thing you could do.  I would like to add 1-2 more examples, preferably ones that involve ruby or python or otherwise more than 1 line of unix magic.

Specifically, examples that use previous output and other advanced features would be very good to include.

As of right now, I have not tested that this script sends data to Instrumental correctly, but, the format is just so simple...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,14 +29,9 @@ By default, Instrumental Daemon will collect metrics on the following server dat
...
-#### Monitoring Services & Other Processes
+## Custom Plugin Scripts
 
-Instrumental Daemon monitors other processes through a powerful plugin system built on binary and shell scripts. Plugin installation and development instructions are listed in [PLUGIN_SCRIPTS.md](PLUGIN_SCRIPTS.md). Existing plugins include:
-
-* [MySQL](examples/mysql)
-* [Mongo](examples/mongo)
-* [Docker](examples/docker)
-* [Redis](examples/redis)
+Instrumental Daemon can monitor arbitrary processes and system events through a pluign scripting system.  Plugin script installation and development instructions are listed in [PLUGIN_SCRIPTS.md](PLUGIN_SCRIPTS.md).
```
Comment (janxious) [inline README.md]: "pluign" -> "plugin"

File: examples/README.md [hunk 2]
```diff
@@ -1,8 +1,31 @@
...
+
+* Queries - `Queries`
+* Bytes Sent  - `Bytes_sent`
+* Bytes Received - `Bytes_received`
+* Connections - `Connections`
+* Slow queries - `Slow_queries`
+
+Additionally, it estimates the number received per second ( `Queries_per_second`, etc. ).
+
+MySQL monitoring functionality is provided by default, but this script demonstrates advanced functionality, including receiving previous script output via STDIN and other advanced features.
```
Comment (tonydewan) [inline examples/README.md]: Perhaps we can make `MySQL monitoring functionality` link to a section in this README about supported services? A section that doesn't exist yet and probably shouldn't be in this PR...

**Comment by tonydewan** [comment]
:+1:

File: examples/README.md [hunk 3]
```diff
@@ -1,8 +1,31 @@
...
 
-* [MySQL](mysql/README.md)
-* [Docker](docker/README.md)
-* [Mongo](mongo/README.md)
-* [Redis](redis/README.md)
+#### Log Size
+
+The [log size](plugin_scripts/log_size.sh) script simply monitors the size of logs in /var/log, so you can see if a log is failing to rotate or growing out of control.  It demonstrates using an executable shell script as a plugin script.
+
+#### ISS Monitor
```
Comment (janxious) [inline examples/README.md]: neat!

**Comment by mediocretes** [comment]
I am running two of these for a minute to collect some displayable data, you can see results here: https://instrumentalapp.com/projects/839?group_id=534&resolution=60&duration=1800&start=1470932649

I started collecting around 12:20 on August 11th, and will collect for an hour or so!

File: examples/README.md (hunk 2 above)
Comment (mediocretes) [inline examples/README.md]: Yes, we should definitely add a link to that section from here when we add that section.

**Comment by janxious** [comment]
I merged this without personally testing. @mediocretes if you didn't test mysql, can you do so? If there are errors, can you fix them?

**Comment by mediocretes** [comment]
I will do that thing.

**Pull request merged**